sequencing/ATACSeq/config/config.txt

```diff
@@ -13,65 +13,73 @@
 ##                                                            ||
 ## ===========================================================##
 
+
+## Example config file ##
+## This config file should be found at the project's folder. The full path to this file should be specified 
+## Please modify as required ##
+
 ## Sequencing data filepaths
 export PROJECT="project name"
-export DATADIR=/lustre/projects/Research_Project-MRC190311/ATACSeq/${PROJECT}
-export CONFIGR=/lustre/projects/Research_Project-MRC190311/ATACSeq/${PROJECT}/config.r
+export MAIN_DIR="/path/to/project/folder/${PROJECT}"
+export CONFIGR="/path/to/R/config/file/${PROJECT}/config.r"
 
-## Specify modules versions
+## Software/script filepaths
+export SCRIPTS_DIR="/path/to/this/scripts/folder"
+export SUB_SCRIPTS_DIR="${SCRIPTS_DIR}/path/to/subcripts/"
+export RSCRIPTS_DIR="${SCRIPTS_DIR}/Rscripts"
```

Comment:
By scripts folder, I presume you mean this specific part of BrainFANS, not BrainFANS' root directory. I assume this because of line 29 here (using `$SCRIPTS_DIR/Rscripts`). If I am correct, could you make it so line 28 is just `$SCRIPTS_DIR/preprocessing` to make it easier on the user?

Reply:
Also, I think I'm having the same problem in my config file templates. Do you think that it users might struggle to find which variables they actually have to change? 
Using the above as an example, it is hopefully obvious that `27` needs to be changed. But then you need to change `28` but not `29`. I think this might confuse some people considering the size of the config file. 

I haven't thought of a good solution for my own set of scripts yet. Maybe a key? I don't know. Just wanted to see if you share that sentiment.

Reply:
The only solution I can think of is to make the user changes as little variables as possible and at the same time expecting a minimum of consistency in folder names (like "Rscripts" or "0_metadata") or other parts of the pipeline. If we don't make that consistency to be expected by the pipeline, then the user will need to change all variables to accommodate to their own data structure. I see it as a never-ending circle.

Reply:
Ah yeah, but what I mean was. If you only expect the user to only change a few lines in the configuration file (out of 30ish). How can we make it obvious to the user which ones they need to change? Obviously they should change the ones that say `path/to/scripts` or something similar in them. But these might not be seen when the whole file is a bunch of path variables. Maybe just putting something in the README as to which variables need to be changed by the user?